File: lib/python/T1562_001.py
TECHNIQUE_ID = "T1562.001"
TACTIC_ID = "TA0005"
TECHNIQUE_NAME = "Impair Defenses: Disable or Modify Tools"

SAFE_PROFILE = {
    "objective": "Assess potential impairment of defenses without altering target state.",
    "risk": "high",
    "steps": [
        "Verify scope authorization and approvals.",
        "Inventory security controls (EDR/AV/agents) from read-only sources.",
        "Review telemetry for unexpected service stops or policy tampering.",
        "Confirm integrity signals (config drift, policy checksum mismatch).",
        "Package evidence for HiL validation and reporting.",
    ],
    "do_not": [
        "Do not disable or modify security tools.",
        "Do not attempt evasion or bypass behaviors.",
    ],
}


def get_assessment_profile():
    return SAFE_PROFILE.copy()

File: lib/rust/T1562_001.rs
pub const TECHNIQUE_ID: &str = "T1562.001";
pub const TACTIC_ID: &str = "TA0005";
pub const TECHNIQUE_NAME: &str = "Impair Defenses: Disable or Modify Tools";

pub fn safe_steps() -> Vec<&'static str> {
    vec![
        "Verify scope authorization and approvals.",
        "Inventory security controls from read-only sources.",
        "Review telemetry for unexpected service stops or policy tampering.",
        "Confirm integrity signals (config drift, checksum mismatch).",
        "Package evidence for HiL validation and reporting.",
    ]
}

pub fn do_not() -> Vec<&'static str> {
    vec![
        "Do not disable or modify security tools.",
        "Do not attempt evasion or bypass behaviors.",
    ]
}
